src/array_api_stubs/_draft/manipulation_functions.py

```diff
@@ -199,6 +199,24 @@ def stack(arrays: Union[Tuple[array, ...], List[array]], /, *, axis: int = 0) ->
     """
 
 
+def unstack(array: array, /, *, axis: int = 0) -> Tuple[array]:
+    """
+    Splits an array in a sequence of arrays along the given axis.
+
+    Parameters
+    ----------
+    array: array
+        input array.
+    axis: int
+        axis along which the array will be split. A valid ``axis`` must be on the interval ``[-N, N)``, where ``N`` is the rank (number of dimensions) of ``x``. If provided an ``axis`` outside of the required interval, the function must raise an exception. Default: ``0``.
+
+    Returns
+    -------
+    out: List[array]
```

Comment:
it's a tuple - please make this consistent with the actual function signature (see my comment higher up)

Reply:
also in the sentence below